uploadNeuroDB/NeuroDB/ImagingUpload.pm

```diff
@@ -410,7 +411,14 @@ Arguments:
 sub PatientNameMatch {
     my $this         = shift;
     my ($dicom_file) = @_;
-    my $cmd          = "dcmdump $dicom_file | grep PatientName";
+
+    my $lookupCenterNameUsing = NeuroDB::DBI::getConfigSetting(
+        $this->{'dbhr'},'lookupCenterNameUsing'
+    );
+
+    my $cmd = sprintf("dcmdump +P %s %s",
+        quotemeta($lookupCenterNameUsing), quotemeta($dicom_file)
```

Comment:
a small comment here:
since the identifier can be patient name or patient ID; then a few lines later, we are specifically telling the user that the patient name can not be extracted ; then 2 lines later, we also say the patient name read the header does not start with ...
do you think it makes sense to make the error message more generic?

Reply:
Very good point!! Thank you for bringing that up. I just made it more generic. :)

Reply:
on another note; the fact u added `sprintf`; does that mean this command will print everytime it is executing?
it did not when tested it, but I was just curious...